src/renderers/dom/shared/hooks/ReactDOMUnknownPropertyHook.js

```diff
@@ -69,7 +69,7 @@ if (__DEV__) {
 
     if (standardName != null) {
       warning(
-        standardName == null,
+        false,
```

Comment:
These are just consistency changes. The original code is kinda odd with two checks.